elephant/asset.py

```diff
@@ -35,7 +35,7 @@
 
 5) Extract sequences of synchronous events associated to each worm
```

Comment:
I am surprised the term 'worm' is still showing up here. I don't think this is introduced anywhere here, so maybe we should replace it by  sequence of synchronous event / asset depending on the context?

Reply:
I remember my confusion in my first month when I looked in asset and one of the first lines of examples says (it still does now!):

```Extract sequences of synchronous events associated to each worm```

I mean, it's not in uppercase :)